### ISSUE TITLE
Huge discrepancy between backtesting and dry run

### ISSUE THREAD

**Issue by petehe** [freqtrade/freqtrade-strategies#237]
## Describe your environment

  * Operating system: _Debian 10___
  * Python Version: _3.8____ (`python -V`)
  * CCXT version: 1.59.2_____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: _2021.10__ (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

When I was running Zeus with 1h timeframe and top 200 binance USDT trading pairs with 40 max open trades as per attached config file, it only generates between 50-100 trades a month, but when I run Zeus in dry run with same setup, it can easily generate 50 trades a day.

### Steps to reproduce:

  1. compare freqtrade trade zeus with 1h timeframe and freqtrade backtest zeus for the month 20211001-20211101 
  2. it shows only around 50 trades a month but in dry run it shows 50 trades a day
  

### Observed Results:

  * What happened? I understand the dry run and backtest are not going to be the same, but the discrepancy is too big to tolerate
  * What did you expect to happen? we have similar number of trades in backtest vs dry run

**Can you please tell me how to diagnose the discrepency between backtest and dry run?**

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
  // paste your log here
  ``
[config_USDT200 .txt](https://github.com/freqtrade/freqtrade/files/7617402/config_USDT200.txt)
`

**Comment by xmatthias** [comment]
While it's a strategy in the freqtrade strategies repository, it has been community provided.
While we try to analyze and vet strategies, we cannot guarantee for it's correctness.

Best verify that the same parameters are used.
What you're describing sounds terribly like a "bad" usage of some indicator (like `qtpylib.vwap`, or the use of `.min()` or `.max()` ) - which would use all available data for it's calculation. 

A quick glance into that strategy also reveals this happening at least twice - which obviously makes every backtest result archived with this strategy invalid.

@mablue as it's your strategy - any comments on this? 
otherwise i think i'll have to remove that strategy, as  in it's current state, it'll never be able to perform even close to the "promised" backtesting results.

**Comment by mablue** [comment]
@xmatthias there is no any bad usage I just have calculated just ichimoku_base and kst_diff but yes maybe one of this indicators have evil inside his codes I will dive more. But yes zeus is profitable but not like when you backtest cuz of I think less market volume and problems like filling orders fast also the stable big stoploss will be very much helpful on some of my strategies for example upper than 0.15

**Comment by xmatthias** [comment]
@mablue unfortunately that's not true, letme point out the faulty part

https://github.com/freqtrade/freqtrade-strategies/blob/7f49409cfdb5173e31c9afe2f2b8303468d4ba83/user_data/strategies/Zeus.py#L99-L104

If you look at these lines, they're scaling using the full dataset (so if you use BTC data from 2018-2021 - you'd get a min of a few 100$ - and a max of 65.000$) - also for data in 2018 - where the bot should not know in the first place that BTC went to 65.000$ (we certainly didn't know back then).

**Comment by mablue** [comment]
> @mablue unfortunately that's not true, letme point out the faulty part
> 
> https://github.com/freqtrade/freqtrade-strategies/blob/7f49409cfdb5173e31c9afe2f2b8303468d4ba83/user_data/strategies/Zeus.py#L99-L104
> 
> If you look at these lines, they're scaling using the full dataset (so if you use BTC data from 2018-2021 - you'd get a min of a few 100$ - and a max of 65.000$) - also for data in 2018 - where the bot should not know in the first place that BTC went to 65.000$ (we certainly didn't know back then).
> 
> You can use this for oscillators (which oscillate between 1 and 0, or 100 and 0) - but not for things like EMA's (ichimoku-base is an EMA, basically), which moves with the price.

Yes you true I really sorry and dont know Why this happen again and again. I will search all normalizations on my strategys and find a way to remove them premently. But I think its the last zombie as well
